templates/ansible/roles/deploy/tasks/main.yml.erb

```diff
@@ -37,30 +37,34 @@
   with_items:
     - config/database.yml
     - config/secrets.yml
-    - <%= "config/#{params[:server_type].split('_')[1]}.rb" %>
+    - config/<%= params[:server_type].split('_')[1] %>.{{ rails_env }}.rb
     - log
     - tmp
     - vendor/bundle
 
-- name: Install bundle
-  command: 'bundle install --deployment --without="development test"'
+- name: Install gems
+  command: /bin/bash -lc 'bundle install --deployment --without="development test"'
   args:
     chdir: '{{ this_release_path }}'
 
 - name: Precompile assets
-  command: bundle exec rake assets:precompile chdir={{ this_release_path }}
+  command: /bin/bach -lc 'rake assets:precompile'
```

Comment:
You're right, the typo was fixed!
As for `bundle exec`, we don't need it if we use `rubygems-bundler` from `rvm`, more details [here](https://rvm.io/integration/bundler)